### PR TITLE
ABL Godunov Hypre test params

### PR DESCRIPTION
**Pull request by PaulMullowney**
Updating these parameters and enabling tests to run.

### CODE REVIEW

**Comment by PaulMullowney** [comment]
@sayerhs you may have to have a look at the solver tolerances to be sure they're appropriate.

**Comment by PaulMullowney** [comment]
Not sure why I deleted:
add_test_re(channel_kwsst_hypre)

I'll fix

File: test/test_files/abl_godunov_hypre/abl_godunov_hypre.i [hunk 1]
```diff
@@ -66,19 +66,44 @@ zhi.type =   "slip_wall"
...
+mac_proj.mg_rtol = 1.0e-8
+mac_proj.mg_atol = 1.0e-13
+
 mac_proj.max_coarsening_level = 0
-mac_proj.bottom_rtol = 1.0e-12
+
+mac_proj.bottom_solver = hypre
+mac_proj.bottom_verbose = 0
+mac_proj.bottom_rtol = 1.0e-4
 mac_proj.bottom_atol = 1.0e-16
```
Comment (sayerhs) [inline test/test_files/abl_godunov_hypre/abl_godunov_hypre.i]:
```suggestion
mac_proj.bottom_rtol = 1.0e-12
mac_proj.bottom_atol = 1.0e-16
```

File: test/test_files/abl_godunov_hypre/abl_godunov_hypre.i [hunk 2]
```diff
@@ -66,19 +66,46 @@ zhi.type =   "slip_wall"
...
 mac_proj.bottom_verbose = 3
-mac_proj.max_coarsening_level = 0
 mac_proj.bottom_rtol = 1.0e-12
 mac_proj.bottom_atol = 1.0e-16
+mac_proj.bottom_maxiter = 100
+mac_proj.hypre_namespace = "mac_proj.hypre"
+mac_proj.hypre.hypre_solver = GMRES
+mac_proj.hypre.hypre_preconditioner = BoomerAMG
+mac_proj.hypre.verbose = 2
+mac_proj.hypre.bamg_verbose = 1
```
Comment (sayerhs) [inline test/test_files/abl_godunov_hypre/abl_godunov_hypre.i:90]:
```suggestion
mac_proj.hypre.verbose = 0
mac_proj.hypre.bamg_verbose = 0
```

File: test/test_files/abl_godunov_hypre/abl_godunov_hypre.i [hunk 3]
```diff
@@ -66,19 +66,46 @@ zhi.type =   "slip_wall"
 zhi.temperature_type = "fixed_gradient"
 zhi.temperature = 0.003 # tracer is used to specify potential temperature gradient
 
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#              HYPRE                    #
+#.......................................#
+
+mac_proj.verbose = 1
+mac_proj.mg_verbose = 1
+mac_proj.mg_cg_verbose = 1
```
Comment (sayerhs) [inline test/test_files/abl_godunov_hypre/abl_godunov_hypre.i:75]:
```suggestion
mac_proj.verbose = 0
```

I recommend we set `verbose = 0` for regression tests, as it is also used as example files for users. `mg_verbose` was replaced with `verbose`.  We removed `mg_cg_verbose` in favor of `bottom_verbose`.

File: test/test_files/abl_godunov_hypre/abl_godunov_hypre.i [hunk 4]
```diff
@@ -66,19 +66,46 @@ zhi.type =   "slip_wall"
...
+mac_proj.mg_rtol = 1.0e-8
+mac_proj.mg_atol = 1.0e-13
+
+mac_proj.max_coarsening_level = 0
+
 mac_proj.bottom_solver = hypre
 mac_proj.bottom_verbose = 3
```
Comment (sayerhs) [inline test/test_files/abl_godunov_hypre/abl_godunov_hypre.i:82]:
```suggestion
mac_proj.bottom_verbose = 0
```

File: test/test_files/abl_godunov_hypre/abl_godunov_hypre.i [hunk 5]
```diff
@@ -66,19 +66,46 @@ zhi.type =   "slip_wall"
...
 
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#              HYPRE                    #
+#.......................................#
+
+mac_proj.verbose = 1
+mac_proj.mg_verbose = 1
+mac_proj.mg_cg_verbose = 1
+mac_proj.mg_rtol = 1.0e-8
+mac_proj.mg_atol = 1.0e-13
```
Comment (sayerhs) [inline test/test_files/abl_godunov_hypre/abl_godunov_hypre.i:77]: For GPU runs, we might need this to be `1e-12`. You can look at CDash results and fine-tune this.

Comment (PaulMullowney) [reply test/test_files/abl_godunov_hypre/abl_godunov_hypre.i:77]: atol or rtol?

Comment (sayerhs) [reply test/test_files/abl_godunov_hypre/abl_godunov_hypre.i:77]: We have had to set `rtol = 1.0e-12` in other tests to get it to consistently pass tests on GPU.

**Comment by michaeljbrazell** [comment]
when running with GPUs on eagle I get this: 

Godunov:
  System                     Iters      Initial residual        Final residual
  ----------------------------------------------------------------------------
  MAC_projection               200         0.00340474802        0.003431631565
  temperature_solve              2       6.862369395e-05       9.947598301e-12
  velocity_solve                 3         0.01210079041       8.206768598e-13
  Nodal_projection               8         0.01413629585       8.806323726e-15

**Comment by michaeljbrazell** [comment]
now it works with the latest cuda/hypre! Thanks Paul I'll merge this.

```Godunov:
  System                     Iters      Initial residual        Final residual
  ----------------------------------------------------------------------------
  MAC_projection                 1       0.0007627115561         4.4136854e-16
  temperature_solve              1       1.920017212e-08       3.353761713e-12
  velocity_solve                 3         0.01182806179       8.100187188e-13
  Nodal_projection               8         0.01371709337       8.870203563e-15

**Pull request merged**